sources/packages/backend/apps/queue-consumers/src/processors/schedulers/notification/process-notifications.scheduler.ts

```diff
@@ -21,20 +18,17 @@ export class ProcessNotificationScheduler extends BaseScheduler<ProcessNotificat
     @InjectQueue(QueueNames.ProcessNotifications)
     protected readonly schedulerQueue: Queue<ProcessNotificationsQueueInDTO>,
     private readonly notificationService: NotificationService,
+    protected readonly queueService: QueueService,
   ) {
-    super(schedulerQueue);
-  }
-
-  /**
-   * TODO:This method will be removed in next PR of #1551.
-   * @returns cron expression.
-   */
-  protected get cronExpression(): string {
-    return "* * * * *";
+    super(schedulerQueue, queueService);
   }
 
-  protected get payload(): ProcessNotificationsQueueInDTO {
-    return { pollingRecordsLimit: PROCESS_NOTIFICATIONS_POLLING_LIMIT };
+  protected async payload(): Promise<ProcessNotificationsQueueInDTO> {
+    const queuePollingRecordsLimit =
+      await this.queueService.getQueueCleanUpPeriod(
+        this.schedulerQueue.name as QueueNames,
+      );
+    return { pollingRecordsLimit: queuePollingRecordsLimit };
```

Comment:
Maybe I am missing something here but why are we getting the `CleanUpPeriod` and assigning it to `queuePollingRecordsLimit`?

Reply:
oh... that was a copy-paste issue. nice catch